### ISSUE TITLE
[NOT MVP] Safari: Tabbing does not work

### ISSUE THREAD

**Issue by steinsebastian** [SAP/spartacus#11570]
It is only possible to tab through menu.
Previous/Next buttons are not reachable.
Some attributes are not reachable:
e.g.
Product Camera -> Group Main Components -> Attribute Camera Body
Product Camera -> Group Insurance and Warranty -> Attribute "Are you a professional photographer?"
Product Camera -> Group Accessories -> no attribute is accessible

(Tested with Safari on Mac)

**Comment by steinsebastian** [comment]
This seems to be related to #11571.

But the issue with the not accessible radiobuttons in Safari exists also in other parts of the SPA UI:
If you proceed to checkout and you are on the "Method of Payment" step you cannot reach the "Payment method" radiobuttons via keyboard tabbing.
Also on the "Delivery Mode" step the "Shipping Method" radiobuttons can't be reached with keyboard tabbing.

**Comment by Changsuwan** [comment]
The tabbing issues in Safari can easily be solved by the browser setting. Initially, the setting in Safari does not allow users to access the elements on the website with tabulator. By following these steps, keyboard users can access every clickable element on the storefront in Safari: Preferences -> Advanced ->  Press tab to highlight each item on a page

![image](https://user-images.githubusercontent.com/79838431/117661202-59888380-b19e-11eb-9795-2a39c2530b40.png)
